.azure-pipelines/release.yml

```diff
@@ -0,0 +1,45 @@
+# Build the release version of the package and publish to artifacts
+
+trigger: none
+
+pool:
+  vmImage: ubuntu-latest
+variables:
+  # Set the release environment variable to build a release version of the wheel
+  ONNX_SCRIPT_RELEASE: 1
+steps:
+  - displayName: 'Use Python 3.10'
+    task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.10'
+  - displayName: Install Python build dependencies
+    script: |
+      python -m pip install --upgrade pip build wheel
+  - displayName: Temporarily rename package to onnx-preview
+    script: |
+      sed -i 's/name = "onnxscript"/name = "onnxscript-preview"/' 'pyproject.toml'
+  - displayName: Build ONNX Script wheel
+    script: |
+      python -m build
+  - task: CopyFiles@2
+    displayName: 'Copy Python Wheel to: $(Build.ArtifactStagingDirectory)'
+    inputs:
+      SourceFolder: 'dist'
+      Contents: '*.whl'
+      TargetFolder: '$(Build.ArtifactStagingDirectory)'
+  # - task: PublishBuildArtifacts@1
+  #   displayName: 'Publish onnxscript'
+  #   inputs:
+  #     ArtifactName: onnxscript
+
+  # Test the wheels. This needs to happen after PublishBuildArtifacts
+  # to avoid interference with the artifacts
```

Comment:
That might make sense for dev wheels, but for released wheel, it seems safer that we require the built wheel to pass all tests and then do upload?

Reply:
Sure. I will make the change in a separate PR.

Reply:
We can trigger the release if the pipeline succeeds